docs/docs/09.2-package-management.md

```diff
@@ -99,7 +99,7 @@ bower install --save react
     <title>Hello React!</title>
     <script src="bower_components/react/react.js"></script>
     <script src="bower_components/react/react-dom.js"></script>
-    <script src="https://unpkg.com/babel-core@5.8.38/browser.min.js"></script>
+    <script src="https://unpkg.com/babel-standalone@6.15.0/babel.min.js"></script>
```

Comment:
In this one, you might want to use babel-standalone from Bower for consistency. Updating the install command to `bower install --save react babel-standalone` should work.